src/stackmap_checker/guard.c

```diff
@@ -0,0 +1,350 @@
+#define UNW_LOCAL_ONLY
```

Comment:
Better add a license to new files.

Reply:
What does `UNW_LOCAL_ONLY` do?

Reply:
It is supposed to run an optimized version of `libunwind`:

```
However, if you tell libunwind that your program only needs local unwinding, then a special implementation can be selected which may run much faster than the generic implementation which supports both kinds of unwinding. To select this optimized version, simply define the macro UNW_LOCAL_ONLY before including the headerfile <libunwind.h>.
```

Reply:
OK. I wonder what "non-local" unwinding would mean?

Reply:
I think anything outside of the current compilation unit is 'non-local', but it's just a guess.